packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -595,6 +626,13 @@ export const TooltipPlugin2 = ({
 
         transform = `translateX(${shiftX}px) ${reflectX} translateY(${shiftY}px) ${reflectY}`;
 
+        // TODO: revisit
+        // if (closestSeriesIdx != null && persistentLinks[closestSeriesIdx]?.length > 0) {
+        //   u.over.style.cursor = 'pointer';
+        // } else {
+        //   u.over.style.cursor = 'default';
+        // }
+
```

Comment:
remove?

Reply:
I left it commented because we need to figure it out, preferably soon. I can remove it for the sake of keeping the code clean.

Reply:
copy/paste this into a follow-up issue, too.